articles/model-anatomy.adoc

```diff
@@ -24,6 +24,13 @@ We deliberately don't use that term in our specifications, as it's so overloaded
 For our purposes, we can always replace "model" with "a bunch of nodes".
 Then the question transforms to: What is a node?
 
+[[notation]]
+== Notation used in this article and diagrams
```

Comment:
I would put this section after the introduction of the node ids and their classifiers, so in between 2.3 `Node classifier` and 2.4 `Node parent`. There it is more logical for the reading flow, and just before it is being used.

Reply:
One day later, I'm not so sure about this paragraph in general -- it moves the needle towards formal spec.

Reply:
moved to appendix